dir.targets

```diff
@@ -81,11 +87,11 @@
          with state.  They were passing because MSTest was serial. -->
 
     <!-- Run tests on the full framework -->
-    <Exec Command="$(PackagesDir)\xunit.runner.console.$(XunitVersion)\tools\xunit.console.x86.exe @(MainAssembly, ' ') -noshadow -parallel none -xml %(MainAssembly.FullPath)_TestResults.xml -html %(MainAssembly.FullPath)_TestResults.html $(XunitOptions) > %(MainAssembly.FullPath)_stdout.txt"
+    <Exec Command="$(PackagesDir)\xunit.runner.console.$(XunitVersion)\tools\xunit.console.exe @(MainAssembly, ' ') $(XunitOptions) -noshadow -parallel none -xml %(MainAssembly.FullPath)_TestResults.xml -html %(MainAssembly.FullPath)_TestResults.html > %(MainAssembly.FullPath)_stdout.txt"
```

Comment:
I thought the x86 version was required for full-framework tests on Windows.  But clearly the tests pass.  Was that a deliberate fix on your part?